engine/src/main/java/com/datatorrent/stram/StramClient.java

```diff
@@ -576,6 +576,12 @@ public void startApplication() throws YarnException, IOException
         vargs.add("-Dlog4j.debug=true");
       }
 
+      String loggerAppender = dag.getValue(LogicalPlan.LOGGER_APPENDER);
+      if (loggerAppender != null) {
+        vargs.add(String.format("-D%s=\"%s\"", LogicalPlan.LOGGER_APPENDER.getName(), loggerAppender));
```

Comment:
Add method (getLongName) to Attributes to convert LOGGER_APPENDER to "apex.logger.appender".